package.json

```diff
@@ -47,10 +47,12 @@
     "eslint-config-webpack": "^1.2.1",
     "eslint-plugin-import": "^2.2.0",
     "extract-text-webpack-plugin": "^3.0.0",
+    "fsevents": "^1.2.4",
```

Comment:
Previously we hadn't depended explicitly on this or nan but I had to specify them to get the right versions installed.